Sources/MuxUploadSDK/Internal Utilities/UploadEvent.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  UploadEvent.swift
+//  
+//
+//  Created by Liam Lindner on 3/22/23.
+//
+
+import Foundation
+import UIKit
+
+struct UploadEvent: Codable {
+    var type = "upload"
+    var startTime: TimeInterval
+    var endTime: TimeInterval
+    var fileSize: UInt64
+    var videoDuration: Double
+
+    var sdkVersion: String
+
+    var osName: String
+    var osVersion: String
+
+    var deviceModel: String
+
+    var appName: String?
+    var appVersion: String?
+
+    var regionCode: String?
+
+    init(startTime: TimeInterval, endTime: TimeInterval, fileSize: UInt64, videoDuration: Double) {
+        let locale = Locale.current
+        let device = UIDevice.current
+
+        self.startTime = startTime
+        self.endTime = endTime
+        self.fileSize = fileSize
+        self.videoDuration = videoDuration
+
+        self.sdkVersion = "0.2.0" // TODO Read from properties
```

Comment:
```suggestion
        self.sdkVersion = "0.2.1" // TODO Read from properties
```

Reply:
This has been fixed to use `Version.versionString` so it no longer has to be updated independently.